docker-compose.yml

```diff
@@ -33,27 +34,27 @@ services:
 
   # https://hub.docker.com/r/localstack/localstack/
   # localstack running a fake S3
-  localstack-s3:
-    image: localstack/localstack:0.10.8
+  localstack:
+    image: localstack/localstack:0.12.2
```

Comment:
I updated localstack and the new version does all the services in a single container with a single port, so I made a bunch of naming and configuration fixes.